general-terms-of-use.md

```diff
@@ -1,48 +1,48 @@
+By accessing or using the Services, checking the box marked [“I Agree”], or by otherwise affirmatively stating your desire to use the Services, you signify that you have read, understood, and agree to be bound by this Agreement and to the collection and use of your information as set forth in the [Clever Privacy Policy](https://clever.com/about/privacy-policy), otherwise you may not use the Services. Because our Services change relatively often, the terms in this Agreement and our Privacy Policy may change too. If you are a School or Developer using the Services, we will notify you via email in advance of any material changes to the terms. Upon making changes, we will update the “Effective Date” found at the top of this page. Your continued use of the Services after any changes constitutes your acceptance of the new terms.
 
 ### SECTION 1. WHAT ARE CLEVER’S SERVICES AND WHO CAN USE THEM?
 
-What do we do? We provide Schools both data integration and single sign-on functionality, making it easier for Schools to choose, deploy, and control third party software applications (“Developers”) used by their teachers and students.
+What do we do? We provide Schools both data integration and single sign-on functionality, making it easier for Schools to choose, deploy, and control software applications made available by Developers that are used by their teachers and students.
 
 You’re allowed to use the Services only if you can form a binding contract with Clever, and only in compliance with this Agreement and all applicable local, state, national, and international laws, rules and regulations. If you’ve been removed from the Services previously, you’re not allowed to use them again.
 
-Subject to the terms and conditions of this Agreement, we grant you a non-exclusive, limited, non-transferable, freely revocable license to use the Services. We reserve all rights not expressly granted to you in the Services and the Clever Content (as defined below). We may terminate this license at any time for any or no reason.
+Subject to the terms and conditions of this Agreement, Subject to the terms and conditions of this Agreement, Clever grants to you permission to access and use the Services. We reserve all rights not expressly granted to you in the Services and the Clever Content (as defined below). We may terminate this license at any time for any or no reason.
```

Comment:
Remove extra "Subject to the terms and conditions of this Agreement," (currently written 2x)